llvm/lib/Target/X86/X86InstrFMA3Info.cpp

```diff
@@ -153,7 +163,8 @@ const X86InstrFMA3Group *llvm::getFMA3Group(unsigned Opcode, uint64_t TSFlags) {
                         ((TSFlags & X86II::EncodingMask) == X86II::EVEX &&
                          ((TSFlags & X86II::OpMapMask) == X86II::T8 ||
                           (TSFlags & X86II::OpMapMask) == X86II::T_MAP6));
-  bool IsFMA3Prefix = (TSFlags & X86II::OpPrefixMask) == X86II::PD;
+  bool IsFMA3Prefix = ((TSFlags & X86II::OpPrefixMask) == X86II::PD) ||
+                      ((TSFlags & X86II::OpPrefixMask) == 0); // X86II::PS
```

Comment:
Remove the outer parentheses.

Reply:
[a6c4fce](https://github.com/llvm/llvm-project/pull/101603/commits/a6c4fce4069fb81ab4a50d825fe6039d45827d63)